tests/datasets/test_entity_validation.py

```diff
@@ -103,242 +105,10 @@
         ),
         id="filtering time with IN is allowed",
     ),
-    pytest.param(
```

Comment:
Yes but I have to completely rewrite these tests since the time condition function is in the SnQL parser now. So I pulled them out so CI could pass and I could get a first pass of reviews while I do that.

Reply:
ok